src/okta-vue.ts

```diff
@@ -53,7 +53,9 @@ export const navigationGuard = async (to: RouteLocationNormalized) => {
     const isAuthenticated = await _oktaAuth.isAuthenticated()
     if (!isAuthenticated) {
       const authState = _oktaAuth.authStateManager.getAuthState()
-      await guardSecureRoute(authState)
+      if (authState) {
```

Comment:
nit: how about handle it in `guardSecureRoute`, we can just do nothing when `authState` is null (app initial state and tokens has not been loaded from storage)

Reply:
Done